src/main/java/com/google/sps/data/Election.java

```diff
@@ -14,70 +14,50 @@
 
 package com.google.sps.data;
 
+import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.Entity;
 import com.google.auto.value.AutoValue;
-import java.util.ArrayList;
 import java.util.HashSet;
+import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 /** A state or national election that will appear on voter ballots */
 @AutoValue
 public abstract class Election {
-  public abstract long getId();
+  public abstract String getId();
 
   public abstract String getName();
 
   public abstract String getDate();
 
   public abstract String getScope();
 
+  // This Election references a collection of Contest entities in Datastore. This HashSet represents
+  // their Key names.
   public abstract HashSet<Long> getContests();
```

Comment:
Should these return Set instead of HashSet? I don't think it's important how the underlying set is implemented from the client's perspective. 

If this is too annoying to cleanup because it's used in too many places, we can leave a TODO.